tests/integration/model/test_models.py

```diff
@@ -148,6 +148,11 @@ def test_export(
             # https://github.com/openvinotoolkit/anomalib/issues/1513
             pytest.skip(f"{model_name} fails to convert to OpenVINO")
 
+        # TODO(samet-akcay, ashwinvaidya17): Fix this test after fixing the issue
+        # https://github.com/openvinotoolkit/anomalib/issues/2047
+        if model_name == "cs_flow" and export_type == ExportType.OPENVINO:
```

Comment:
From the logs, it looks like that the torch JIT fails. Probably `wandb` is changing some hook. Either case, I think skipping the torch export rather than openvino might make the test pass. However, I am not sure if onnx export is affected as well.